packages/flutter_webrtc/flutter-webrtc-demo/pubspec.yaml

```diff
@@ -0,0 +1,30 @@
+name: flutter_webrtc_demo
```

Comment:
How is this package different from flutter_webrtc_example? If you really want to have two example apps for a single plugin, consider putting both directories in the `example` directory ([example](https://github.com/flutter-tizen/plugins/tree/master/packages/tizen_rpc_port/example)).

Reply:
flutter_webrtc_demo : Copy from https://github.com/flutter-webrtc/flutter-webrtc-demo, we use this package for development.
I have moved two directories to example directory.